docs/content/guide/migration.ngdoc

```diff
@@ -88,7 +88,7 @@ commits for more info.
 
 - **jqLite** is more aligned to jQuery 3, which required the following changes
   (see [details](guide/migration#migrate1.5to1.6-ng-misc-jqLite) below):
-  - Keys passed to `.data()` and `.css()` are now camelCased in the same way as jQuery does it.
+  - Keys passed to `.data()` and `.css()` are now camelCased in the same way as jQuery.
```

Comment:
While I agree the original sentence is not ideal, I find the new version ambiguous. It can be interpreted as:

> [...] are now camelCased in the same way as jQuery is camelCased